hack/local-down-karmada.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage() {
+    echo "Usage:"
+    echo "hack/local-down-karmada.sh [-r] [-h]"
+    echo "Description:"
+    echo "r, remove-images, the local images needs to be removed"
+    echo "h, help, print help information"
```

Comment:
```suggestion
    echo "    r, remove-images: the local images needs to be removed"
    echo "    h, help: print help information"
```

Reply:
the `remove-images` and `help` options really work?

Reply:
> the `remove-images` and `help` options really work?

@ikaven1024 
```sh
➜  karmada git:(feature/localdown) hack/local-down-karmada.sh

Start removing kind clusters
Deleting cluster "karmada-host" ...
Deleting cluster "member1" ...
Deleting cluster "member2" ...
Deleting cluster "member3" ...
Remove kind clusters successfully.

Start removing kubeconfig
Remove kubeconfig successfully.

Start removing images
Skip, No images need to be removed

Local Karmada is removed successfully.
➜  karmada git:(feature/localdown) hack/local-down-karmada.sh -r

Start removing kind clusters
Deleting cluster "karmada-host" ...
Deleting cluster "member1" ...
Deleting cluster "member2" ...
Deleting cluster "member3" ...
Remove kind clusters successfully.

Start removing kubeconfig
Remove kubeconfig successfully.

Start removing images
Error: No such image: docker.io/karmada/karmada-controller-manager:latest
Error: No such image: docker.io/karmada/karmada-scheduler:latest
Error: No such image: docker.io/karmada/karmada-descheduler:latest
Error: No such image: docker.io/karmada/karmada-webhook:latest
Error: No such image: docker.io/karmada/karmada-scheduler-estimator:latest
Error: No such image: docker.io/karmada/karmada-aggregated-apiserver:latest
Error: No such image: docker.io/karmada/karmada-search:latest
Remove images successfully.

Local Karmada is removed successfully.
➜  karmada git:(feature/localdown) hack/local-down-karmada.sh -h
Usage:
    hack/local-down-karmada.sh [-r] [-h]
Args:
    r, remove-images: the local images needs to be removed
    h, help: print help information
➜  karmada git:(feature/localdown) hack/local-down-karmada.sh -d
hack/local-down-karmada.sh: 非法的选项 -- d
Usage:
    hack/local-down-karmada.sh [-r] [-h]
Args:
    r, remove-images: the local images needs to be removed
    h, help: print help information
```